Intersect.Client/Entities/Player.cs

```diff
@@ -2372,6 +2376,39 @@ public void DrawOverheadInfoOnHover()
             }
         }
 
+        protected override bool ShouldDrawHpBar
+        {
+            get
+            {
+                if (!ShouldNotDrawHpBar || IsHovered)
+                {
+                    return true;
+                }
+
+                if (this.Id == Globals.Me.Id)
```

Comment:
`this.Id` -> `Id`

Reply:
Hmm but, don't we have to check against "my ID" in order to tell filter "my character" from other players (?)

`MyOverheadHpBar =/= PlayerOverheadHpBar`